embedded-graphics/src/fonts/font6x8.rs

```diff
@@ -19,11 +19,14 @@ pub struct Font6x8<'a> {
 
     /// Text to draw
     text: &'a str,
+
+    /// Color
```

Comment:
I think it's worth noting that this is the font _fill_ colo(u)r in this doc comment. Same applies to the other fonts too :)